07-Publishing-Repositories.Rmd

```diff
@@ -4,7 +4,46 @@ editor_options:
     wrap: 80
 ---
 
-# Publishing Repositories
+# Archiving Data and Code
+
+We archive data and code through a number of pathways. Archiving our code and
+data make downstream analyses reproducible, is a great way to share data, and
+also provide the ability for citation through a stable DOI. Generally speaking,
+it's good practice to get a DOI for any data before sharing it, as this helps
+with data provenance, additionally, most journals now require that all code and
+data are archived using a DOI. The flowchart below will guide you in the
+preferred method of archiving.
+
+![](images/Data_Code_Archiving.png)
```

Comment:
I know we did this together but I now think the arrow after the "Your data and code should be archived separately" should connect to "Document your data with metadata..." instead of to the top of the decision tree.

Reply:
I think we should REMOVE the arrow back to the top. :)